src/components/InputForm.js

```diff
@@ -38,13 +39,15 @@ const InputBooks = () => {
   };
 
   return (
-    <div>
-      <form onSubmit={createNewBook}>
-        <input type="text" name="Book" placeholder="Book" required value={title} onChange={BookInput} />
+    <footer className="footer">
+      <h1 id="add-book">ADD NEW BOOK</h1>
+      <form className="form" onSubmit={createNewBook}>
+        <input type="text" name="Book" placeholder="Book title" required value={title} onChange={BookInput} />
         <br />
         <input type="text" name="author" placeholder="Written By" required value={author} onChange={authorInput} />
         <br />
-        <select name="category" id="category" onChange={anyName}>
+        <select name="category" id="category" onChange={anyName} className="d-option">
+          <option>Category</option>
```

Comment:
- [ ] Kindly increase the input boxes sizes for your inputs boxes match the desired design that is provided in Zeplin design.

**These are how your input boxes looks like now:**

![image](https://user-images.githubusercontent.com/86318284/202714208-623f0fdf-823a-4a94-9e80-945f227da070.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/202714276-89b5c8a8-5c1a-4ecb-a6a3-0d4f64118847.png)